source/client.ts

```diff
@@ -199,6 +199,19 @@ function formatDocument(
   return formatDocumentRange(languageId, channel, document, range, token)
 }
 
+function expandTemplate(
+  template: string,
+  replacements: { [key: string]: string }
+): string {
+  return template
+    .replace(/(?<!\\)\$\{([a-z]+)\}/, (_, key) => {
```

Comment:
This isn't quite right. If the template was `"\\${x}"`, then it should be expanded. If there's an _even_ number (including zero) of back slashes before the dollar sign, then expansion should happen. Otherwise it shouldn't.